core/deps/imgui/imgui.cpp

```diff
@@ -1,4 +1,4 @@
-// dear imgui, v1.90.8
+// dear imgui, v1.92.1
```

Comment:
Files in `core/deps/imgui` have 100755 permissions instead of 100644, can you fix that?